ui5.yaml

```diff
@@ -17,10 +17,10 @@ builder:
           afterTask: replaceVersion
           configuration: &transpileConfig
               removeConsoleStatements: true
-              transpileAsync: true
+              transpileAsync: false
               transpileTypeScript: true
         - name: ui5-tooling-modules-task
-          afterTask: replaceVersion
+          afterTask: ui5-tooling-transpile-task
```

Comment:
the order of the plugin is important - transpiling must happen first. but as said - I can check tomorrow.

Reply:
yeah, i think that´s the important part

Reply:
tried again, thats the only change needed